### DISCUSSION TITLE
Maintainer Applications

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#123] [Announcements]
Hello everyone!

Recently, we have been getting tons of activity, I would like to thank you all, all the people who have been helping others, and everyone who has learned something.

In order to decrease the load on me moderating all of the activity, I am looking for Maintainers that can help moderate the discussions. Please comment below on 1. Why should you be a maintainer 2. What will you do as a maintainer.

**Comment by ghost** [comment]
Hello, I would like to be a maintainer because I have moderate knowledge of how the exploit works, and contribute. I will answer questions and if you're ok, delete spammers. I will also post updates in threads

**Comment by Sxri** [comment]
Hello, I'd like to be a maintainer because I enjoy helping others and have knowledge on tech, especially school cheats. I've made multiple cheats and exploits in the past, however, most of them have been patched as well. As a maintainer I will answer questions and concerns consistently with the most accurate reply, along with suggesting or giving the users safe websites, wiki's, and more to help support them in finding their answer to their question. Along with that, if it ever gets to the point of spamming, I will make sure to delete spam comments or unnecessary comments regarding stuff off topic, ex: self advertising, spam links, etc. 

I hope this application fits your needs! 

Best regards,
lurk/sxri

**Comment by The-Man-Is-Me** [comment]
Why I should be a monorator.
1.Im cool
2.On a more serious note I really just want to help. 
3.I have a limited background in how this exploit works and other available exploits
4.I am just your average person. I can see where other people are coming from and how to best help them.
5.Also, it would just be kinda fun.
What I would do.
1.Answer questions
2.Keep people off your back
3.Make sure people are helping each other
Peace, The Man

**Comment by ehunt34** [comment]
I would like to be a maintainer. I have very good knowledge of this exploit and ChromeOS/ChromiumOS/Linux in general. I mainly have experience with Arch. My current goal is to find a way to activate Developer Mode on a managed device to install custom firmware to boot other Linux distros. I can solve issues, answer questions, and add further exploits to the chain that starts with LTBeef/Ingot.

**Reply by ghost** [reply]
Tell me, how does the exploit work, and what part of the exploit does what?

**Reply by ghost** [reply]
you also seem to not have made any original repos. That's usually a red flag, since most people with no idea what they're doing fork everything and don't commit a single time. Well, I'll see. anyways...

**Reply by ehunt34** [reply]
I know how the exploit works (well, worked). It was essentially just Javascript code that told the Chrome Web Store to turn on/off extensions. It asks what extensions to disable, either through ID or GUI depending on which version, then tells the Web Store to turn off that extension. The Web Store, being a Chrome component extension, had power over all extensions, including force installed ones. The Web Store did not have a check to make sure it was installed by management, so it listened to the code. This was fixed in both 106 and LTS 102 by adding that check, which fixed the exploit.

My plan for installing Arch on a managed device would be to downgrade to 100, (not 101 because I couldn't find that image for jacuzzi on chrome100.dev) use the bash exploit to remove enrollment, activate Developer Mode, flash a custom firmware, likely either Mr Chromebox's or a custom compiled version of coreboot, then reenroll ChromeOS. What I like in the exploits is a way to hide or turn them off, like with LTBeef/Ingot with turning extensions back on, or this with still having ChromeOS. The only issue is still being able to use other OS's without Developer Mode, which I assume needs changes to the coreboot firmware. After that, I would be able to use a Usb Drive with Arch installed on it to use Arch freely.

I will admit, I don't have any personal github projects other than a private dotfiles repository, so I have that against me. I do know how most of my forks work, I just fork them so I can have them personally either to just have my own copy or to make fixes or changes (rarely).

**Reply by ehunt34** [reply]
Another thing I forgot to mention is I only use exploits like this when I don't have anything else to do. I'm not one of those people who use these to constantly avoid work. That is why I still like to have standard, monitored, ChromeOS, because I still do school work on that. It just pisses me off to no end when they block things I would do during free time just because some people didn't do their work because of it. My ultimate goal is to learn Computer Science in collage, so I obviously still need to do school work.

**Reply by ghost** [reply]
Seems kinda sketchy not gonna lie... I mean, I did post a dumm'd- down explanation on the github so you could have just copy pasted, and I checked your profile only to see that you hadn't commited a single time. The fact that you don't want to fix the extension but to do a VERY RISKY maneuver also shows recklessness and desperation, which is a giant IF when considering these. I mean, most people here are like 12 and just want to play coolmath or whatever. I'm personally against the whole downgrade-system-swap thing you guys want to do, since most likely you guys aren't the right people to handle that type of thing. Another giant red flag I'm considering is that you seem to be one of  those "copy unblock repo" throwaway accounts that are just used to get unblockers, which in my experience are very risky since we don't know who you are and it's very easy to do something bad and get away with it.
PS: you spelled college wrong

**Reply by ghost** [reply]
@3kh0

**Reply by The-Man-Is-Me** [reply]
@yeeteeyt you lit him up a bit :)

**Reply by ghost** [reply]
@The-Man-Is-Me too much?

**Comment by S-PScripts** [comment]
im a sigma
